.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  release:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        scala_version: &scala_versions
+          - 2.12.12
+          - 2.13.4
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with: &cache_options
+          path: |
+            ~/.ivy2/cache
+            ~/.sbt
```

Comment:
do we want the full `~/.sbt` or just `~/.sbt/boot` ?

Reply:
IDK, I was just using what I saw in an example.

Reply:
sgtm; asking cause the cache we have on travis uses the latter